src/openforms/forms/admin/tasks.py

```diff
@@ -102,3 +103,33 @@ def process_forms_import(import_file: str, user_id: int) -> None:
 def clear_forms_export():
     logger.debug("Clearing old export files")
     call_command("delete_export_files")
+
+
+@app.task()
+def activate_forms():
+    """Activate all the forms that should be activated by the specific date and time."""
+    now = timezone.now().minute
+    forms = Form.objects.filter(
+        active=False, _is_deleted=False, activate_on__minute=now
+    )
```

Comment:
Shouldn't this also check that the date is correct?

Reply:
You are totally right, thanks for this!